pdc_client/__init__.py

```diff
@@ -42,19 +42,8 @@ def read_config_file(server_alias):
 
 def get_paged(res, **kwargs):
     """
-    This call is equivalent to ``res(**kwargs)``, only it retrieves all pages
-    and returns the results joined into a single iterable. The advantage over
-    retrieving everything at once is that the result can be consumed
-    immediately.
-
-    :param res:     what resource to connect to
-    :param kwargs:  filters to be used
-
-    ::
-
-        # Example: Iterate over all active releases
-        for release in get_paged(client['releases']._, active=True):
-            ...
+    Warning: This method have been removed to PDCClient class;
+    and then will be deleted in next release.
```

Comment:
The docstring will not rendered, so maybe we could add one [PendingDeprecationWarning](https://docs.python.org/2.7/library/exceptions.html#exceptions.PendingDeprecationWarning) as well.